rosetta-testnet/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '3'
+
+services:
+ observer-0:
+   image: elrondnetwork/${NODE_TAG}
+   ports:
+     - "8080:8080"
+     - "10000:37373"
+   environment:
+     - SHARD=0
+   volumes:
+     - ~/MyObservingSquadTestnet/node-0/db:/go/elrond-go/cmd/node/db
+     - ~/MyObservingSquadTestnet/node-0/logs:/go/elrond-go/cmd/node/logs
+     - ~/MyObservingSquadTestnet/node-0/config:/config
+   command: --destination-shard-as-observer=${SHARD_0} --validator-key-pem-file=/config/observerKey_${SHARD_0}.pem --display-name="${DISPLAY_NAME_0}"
+   networks:
+     elrond-squad:
+       ipv4_address: 10.0.0.6
+
+ observer-1:
+   image: elrondnetwork/${NODE_TAG}
+   ports:
+     - "8081:8080"
+     - "10001:37373"
+   environment:
+     - SHARD=1 
+   volumes:
+     - ~/MyObservingSquadTestnet/node-1/db:/go/elrond-go/cmd/node/db
+     - ~/MyObservingSquadTestnet/node-1/logs:/go/elrond-go/cmd/node/logs
+     - ~/MyObservingSquadTestnet/node-1/config:/config
+   command: --destination-shard-as-observer=${SHARD_1} --validator-key-pem-file=/config/observerKey_${SHARD_1}.pem --display-name="${DISPLAY_NAME_1}"
+   networks:
+     elrond-squad:
+       ipv4_address: 10.0.0.5
+ observer-2:
+   image: elrondnetwork/${NODE_TAG}
+   ports:
+    - "8082:8080" 
+    - "10002:37373"
+   environment:
+     - SHARD=2
+   volumes:
+     - ~/MyObservingSquadTestnet/node-2/db:/go/elrond-go/cmd/node/db
+     - ~/MyObservingSquadTestnet/node-2/logs:/go/elrond-go/cmd/node/logs
+     - ~/MyObservingSquadTestnet/node-2/config:/config
+   command: --destination-shard-as-observer=${SHARD_2} --validator-key-pem-file=/config/observerKey_${SHARD_2}.pem --display-name="${DISPLAY_NAME_2}"
+   networks:
+     elrond-squad:
+       ipv4_address: 10.0.0.4
+
+ observer-meta:
+   image: elrondnetwork/${NODE_TAG}
+   ports:
+     - "8083:8080"
+     - "10003:37373"
+   environment:
+     - SHARD=metachain
+   volumes:
+     - ~/MyObservingSquadTestnet/node-metachain/db:/go/elrond-go/cmd/node/db
+     - ~/MyObservingSquadTestnet/node-metachain/logs:/go/elrond-go/cmd/node/logs
+     - ~/MyObservingSquadTestnet/node-metachain/config:/config
+   command: --destination-shard-as-observer=${SHARD_M} --validator-key-pem-file=/config/observerKey_${SHARD_M}.pem --display-name="${DISPLAY_NAME_M}"
+   networks:
+     elrond-squad:
+       ipv4_address: 10.0.0.3
+
+ proxy:
+   image: elrondnetwork/${PROXY_TAG}
+   ports:
+     - "8079:8079"
+   environment:
+     - IP=10.0.0.2
+   networks:
+     elrond-squad:
+       ipv4_address: 10.0.0.2
+
+networks:
+  elrond-squad:
+    ipam:
+      config:
+        - subnet: 10.0.0.0/24
```

Comment:
I wonder if we should add the subnet as an env variable. @AdoAdoAdo, @valentin-lup? But not now, in the future.